evalml/pipelines/components/transformers/imputers/imputer.py

```diff
@@ -161,7 +161,8 @@ def transform(self, X, y=None):
         if self._numeric_cols is not None and len(self._numeric_cols) > 0:
             X_numeric = X.ww[self._numeric_cols.tolist()]
             imputed = self._numeric_imputer.transform(X_numeric)
-            X_no_all_null[X_numeric.columns] = imputed
+            for numeric_col in X_numeric.columns:
```

Comment:
I'm not sure whether we want to file an issue with Woodwork for the ability to do this?  Basically I wanted to use the woodwork table accessor to assign a handful of columns to an existing dataframe.